### PR TITLE
[C-2905] Update Text types and props to camelCase

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Updating the types and props to be camelCase for consistency and dev ergonomics

### Dragons

N/A

### How Has This Been Tested?

Manually tested

### How will this change be monitored?

N/A

### Feature Flags ###

N/A

### CODE REVIEW

**Review by schottra — APPROVED** [review]
Thank you 💯 
If it's not inconvenient, there are a handful more usages that just got merged and need to be updated.

I'm happy to do it in a separate PR after you merge this if you're busy!

**Comment by Kyle-Shanks** [comment]
> 

As i see them, I'll update those too 👍

**Review by dylanjeffers — APPROVED** [review]
yupppp

**Pull request merged**
